resources/main.qml

```diff
@@ -167,11 +167,38 @@ ApplicationWindow {
             }
         }
 
+        DelegateModel {
+            id: displayDelegateModel
+            delegate: pageDelegate
+            model: pageModel
+
+            groups: [
+                DelegateModelGroup {
+                    id: configuredPages
+                    //includeByDefault: true
+                    name: "configured"
+                    property var model_index_map: []
```

Comment:
Please remove any reference to `model_index_map`. This variable is not used in qtphy.